tests/test.py

```diff
@@ -4689,6 +4759,24 @@ def executePreparedStatement(self, query, args=None):
         ).execute_prepared_statement(query, args=args)
 
 
+class TestMysqlConnectorCBinaryPreparedStatement(BasePrepareStatementMixin, BaseTestCase):
```

Comment:
can you add some comment with description what we expected this connector will do? it will help to understand at the future if this test will fail after updating driver or something else. and explain what we expected and should reproduce. because after a year if this test will fail, we will not remember what behavior we expected and what to do to back it again